libs/ha/ha_common_libs_k8s.py

```diff
@@ -1907,3 +1929,45 @@ def form_endpoint_port(pod_obj, pod_list, port_name="rgw-https"):
             pod_ep_dict[pod] = ip_port
 
         return pod_ep_dict
+
+    def create_iam_user_with_header(self, i_d, header):
+        """
+        Function create IAM user with give header info.
+        :param i_d: Int count number for IAM user name creation
+        :param header: Existing header to use for IAM user creation post request
+        :return: None if IAM user REST req fails or Dict response for IAM user successful creation
+        """
+        user = None
+        payload = {}
+        name = f"ha_iam_{i_d}_{time.perf_counter_ns()}"
```

Comment:
Lets make this method generic, we can create users in loop, loop for num_users.

Reply:
> Lets make this method generic, we can create users in loop, loop for num_users.

We have to update main loop logic also for that, which will take time. 
And do we really need this. This is specific to control pod test only. Why we want to make it generic

Reply:
Please make changes as discussed